docs/assets/scss/_variables_project.scss

```diff
@@ -0,0 +1 @@
+$light: #fff;
```

Comment:
where else is this used? - should we maybe do this more specific and only for this kind of box, rather than setting this global?

Reply:
It was not used anywhere else but I just changed it. I used `-bg-white` to set the bg color this time.